tests/test_kldiv.py

```diff
@@ -150,3 +177,37 @@ def test_unknown_size_inputs():
     unknown_shape = gauss_kl(mu, sqrt)
 
     np.testing.assert_allclose(known_shape, unknown_shape)
+
+
+@pytest.mark.parametrize('white', [True, False])
+def test_q_sqrt_constraints(inducing_points, kernel, mu, white):
```

Comment:
Would it be worth unifying the different ways of specifying input arrays (pytest fixtures vs Datum class)? I don't have any preference either way...

Reply:
pytests fixtures have the advantage of only running once then making the most of pytests in built caching. In general we should move to fixtures, though I also want to look at different ways you can sort of parameterize fixtures. (ie run with multiple diff fixtures)